Source/GTMSessionFetcher.m

```diff
@@ -1121,16 +1121,14 @@ - (void)simulateDataCallbacksForTestBlockWithBodyData:(NSData *GTM_NULLABLE_TYPE
 
     // Simulate receipt of redirection.
     if (willRedirectBlock) {
-      [self invokeOnCallbackUnsynchronizedQueueAfterUserStopped:YES
-                                                          block:^{
-                                                            willRedirectBlock(
-                                                                (NSHTTPURLResponse *)response,
-                                                                self->_request,
-                                                                ^(NSURLRequest *redirectRequest){
-                                                                    // For simulation, we'll assume
-                                                                    // the app will just continue.
-                                                                });
-                                                          }];
+      __auto_type block = ^{
```

Comment:
Is __auto_type support in all our Xcode versions or should we bump that first?

Reply:
Good point, I'll take that on first.